client/app/rules/rules.run.js

```diff
@@ -1,9 +1,9 @@
 /* @ngInject */
 function redirectToHabanero ($rootScope, $state, $window, CONFIG) {
   $rootScope.$on('$stateChangeStart', (event, to, params) => { // eslint-disable-line
-    if (CONFIG.habanero.host && to.name.startsWith('rules')) {
+    if (CONFIG.habanero.embedded && to.name.startsWith('rules')) {
       event.preventDefault()
-      $window.open(CONFIG.habanero.host, '_blank')
+      $window.open('/gotoHabanero', '_blank')
```

Comment:
It think `habanero` or `goto-habanero` would be better. I rarely see camelCase endpoints.